### DISCUSSION TITLE
Current National or Regional Estimates of Rt or Re for COVID-19?

### DISCUSSION THREAD

**Discussion by william-silversmith** [epiforecasts/EpiNow2#497] [Q&A]
Hi! It has become more and more difficult to find a current published national (let alone regional) estimate of Rt or Re for COVID-19. I was wondering if anyone has been routinely doing so using EpiNow2 (at least weekly). I had been hoping to use an estimate for some visualizations. Do you know if this is available? This is a bit outside my field, so apologies if I'm missing something obvious (such as the data quality has deteriorated so much that this is impossible).

**Comment by seabbs** [comment]
Hi @william-silversmith,

Unfortunately, as a dev team we haven't had the capacity to produce these for some time and haven't been able to get resources to increase that capacity. 

As far I am aware there are very few/no pipelines still producing useful Re estimates which is a shame. That being said, at least in the US ,  I think there are a few initiatives to turn that around that should (fingers crossed) come online in the next few months producing these at the state level. When one of those pops up I will circle back here and post it.

**Reply by william-silversmith** [reply]
Thank you, this is very helpful to know! I'll keep my eyes peeled then. Really appreciate you taking the time to answer.
